CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+##Supported Release 2.2.1
+###Summary
+
+Small release for support of newer Puppet versions.
```

Comment:
This is by no means an adequate change log. We should actually state what's changed/fixed. The generic "bug fixes and improvement" changelogs are incredibly infuriating for anyone that actually needs to figure out what changes made it in when.